lib/utils/privilegeLevelUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { PrivilegeLevel } from '@prisma/client';
+import { Session } from 'next-auth';
+import { PRIVILEGE_LEVEL_ORDERING } from '../constants';
```

Comment:
This constant should be in this file. The point of getIndexOfPrivilegeLevel is so you don't have to export the ordering to any other part of the app

Reply:
What if we want to use PrivilegeLevelOrdering in places other than privilegeLevelUtils?

I feel like having one file for all constants may be helpful so that someone doesn't have to go look for where certain constants are